sui_types/Cargo.toml

```diff
@@ -15,6 +15,7 @@ once_cell = "1.10.0"
 rand = "0.7.3"
 serde = { version = "1.0.137", features = ["derive"] }
 ed25519-dalek = { version = "1.0.1", features = ["batch", "serde"] }
+curve25519-dalek = { version = "3", default-features = false, features = ["serde"] }
```

Comment:
Is this PR already implementing a non-naive checkpointing strategy? (cfr https://www.notion.so/mystenlabs/Sui-Checkpoints-from-Naive-to-Simple-to-Efficient-422ab63300a14d61825f169b830fd0d3)

Reply:
Yes, the naive scheme is to just illustrate the approach. Waypoints and diffs, and combinations of diffs as fragments are included here. So we do full spanning tree checkpointing.